test/vdb/entry.cpp

```diff
@@ -0,0 +1,23 @@
+#include "pms-utils/vdb/vdb.hpp"
+
+#include <filesystem>
+#include <iostream>
+#include <ostream>
+
+// NOLINTBEGIN(bugprone-exception-escape)
+
```

Comment:
just make this a `// NOLINTNEXTLINE(bugprone-exception-escape)` above the `main` decl, like all the other  tests do?